porter/v1/cluster_control_plane.proto

```diff
@@ -1100,6 +1119,9 @@ service ClusterControlPlaneService {
   // LatestEnvGroupWithVariables returns the latest env group and variables for a given env group name in the given deployment target
   rpc LatestEnvGroupWithVariables(LatestEnvGroupWithVariablesRequest) returns (LatestEnvGroupWithVariablesResponse) {}
 
+  // AppEnvVariables returns the latest env variables for a given app in the given deployment target
+  rpc AppEnvVariables(AppEnvVariablesRequest) returns (AppEnvVariablesResponse) {}
```

Comment:
if this returns secrets mention in the comment that it returns sensitive information